graphql/schema.graphql

```diff
@@ -495,6 +498,34 @@ type SiteBanner {
   theme: String!
 }
 
+type Host{
+  id: String!    
+	hostURL: String!    
+	distro: DistroInfo     
+	startedBy: String!   
+	provider: String!    
+	user: String!    
+	status: String!   
+	runningTask: TaskInfo  
+	displayName: String!
+  LastCommunicationTime: Time! 
+}
+
+
+type DistroInfo {
+  id: String! 
+	provider: String! 
+	imageId: String! 
+}
+
+type TaskInfo {
+  id: String!    
+	name: String!   
+	dispatchTime: Time! 
+	versionId: String!   
+	buildId: String! 
```

Comment:
what are all these used for?

Reply:
we only need id and name. removing the rest.